.circleci/config.yml

```diff
@@ -52,14 +52,15 @@ commands:
       directory:
         type: string
     steps:
+
       - run: 
           name: Replace API key
           command: bundle exec fastlane replace_api_key_integration_tests
       
       - run:
           name: Run tests
           working_directory: << parameters.directory >>
-          command: bundle exec fastlane scan
+          command: bundle exec fastlane test
```

Comment:
Do we need this change?  I thought `scan` is what's used to run tests

Reply:
scan is the default tool, but we have our own lane that we use for testing more than one platform at a time (tvOS + iOS). See [this pr](https://github.com/RevenueCat/purchases-ios/pull/729)